pages/entity/_type/_.vue

```diff
@@ -125,7 +126,10 @@
         return this.editorialDepiction ? this.$t('goToRecord') : this.$t('entityDepictionCredit');
       },
       description() {
-        return this.editorialDescription ? this.editorialDescription : entities.getEntityDescription(this.entity);
+        return this.editorialDescription ? { values: [this.editorialDescription], code: null } : entities.getEntityDescription(this.entity, this.$i18n.locale);
+      },
+      descriptionText() {
+        return (this.description && this.description.values.length >= 1) ? this.description.values[0] : `${this.title} ${this.$t('entity')}`;
```

Comment:
The fallback for description meta tags when no actual description is present is not a useful, i.e. descriptive, one. Let's omit those tags when there is no description.